R/run_fims_age.R

```diff
@@ -0,0 +1,50 @@
+run_fims_age <- function(data = data_mile1) {
```

Comment:
Add roxygen documentation. After adding, run devtools::document() to update Rmd files and Namespace. If you get an error that the .o file is to big, run this line of code first: `withr::local_options(pkg.build_extra_flags = FALSE)`, which depends on the R package, pkgbuild being installed locally.

Reply:
After discussing with @kellijohnson-NOAA, we think it is better to not include run_fims_age function in the pull request since the function is half baked.

Reply:
Sounds good.